src/Utilities.mss

```diff
@@ -234,25 +234,84 @@ function TupletsEqual (t, t2) {
 
 }  //$end
 
-function IsLastNoteInTuplet (bobj) {
+function CountTupletsEndingAtNoteRest(noteRest) {
     //$module(Utilities.mss)
-    next_obj = bobj.NextItem(bobj.VoiceNumber, 'NoteRest');
-
-    if (next_obj = null)
+    tuplet = noteRest.ParentTupletIfAny;
+  
+    if (tuplet = null)
     {
-        return true;
+        return 0;
```

Comment:
This function counts how many tuplets end at a specific `NoteRest`.  If the `NoteRest` is not part of a tuplet, no (i.e. 0) tuplets end here.